File: Calculator.java
import java.util.Scanner;
public class Calculator {
    public static void main(String[] args) {
        Scanner input=new Scanner(System.in);
        float n1,n2;
        System.out.print("Enter number 1: ");
        n1=input.nextFloat();
        System.out.print("Enter number 2: ");
        n2=input.nextFloat();
        int choice;
        System.out.println("1.Addition,2.Substraction,3.Multiplication,4.Division.");
        System.out.println("Enter any choice: ");
        choice=input.nextInt();
        if(choice==1){
            System.out.println(n1+"+"+n2+"="+(n1+n2));
            System.out.println("the output"+(n1+n2)+"is printed");
        }
        else if(choice==2)
        {
            System.out.println(n1+"-"+n2+"="+(n1-n2));
            System.out.println("the output"+(n1-n2)+"is printed");
        }
        else if(choice==3){
            System.out.println(n1+"*"+n2+"="+(n1*n2));
            System.out.println("the output"+(n1*n2)+"is printed");
        }
        else if(choice==4){
            System.out.println(n1+"/"+n2+"="+(n1/n2));
            System.out.println("the output"+(n1/n2)+"is printed");
        }
        else{
            System.out.println("you have entered wrong choice please check the above menu and continue!!");
        }
        input.close();
    }
}

File: calculator.cpp
#include<iostream>
#include<string>
using namespace std;
int main()
{
    cout<<"1.addition\n2.substraction\n3.multiplication\n4.division\n";
    int choice;
    cin>>choice;
    cout<<"enter two numbers\n";
    float num1,num2;
    cin>>num1>>num2;
    switch(choice)
    {
        case 1:
                cout<<"addition of "<<num1<< " and " <<num2<<" is " <<num1+num2<<endl;
                break;
        case 2:
                cout<<"substraction of "<<num1<< " and " <<num2<<" is " <<num1-num2<<endl;
                break;
        case 3:
                cout<<"multiplication of "<<num1<< " and " <<num2<<" is " <<num1*num2<<endl;
                break;
        case 4:
                cout<<"division of "<<num1<< " and " <<num2<<"is " <<num1/num2<<endl;
                break;
        default : cout<<"enter right choice !!";
    }

}
